e2e/pipelines/test_standard_pipelines.py

```diff
@@ -0,0 +1,220 @@
+                params:
+                sql_url: sqlite:///{tmp_path}/faiss_document_store.db
+            - name: PDFConverter
+                type: PDFToTextConverter
+                params:
+                remove_numeric_tables: false
+            - name: TextConverter
+                type: TextConverter
+            - name: Preprocessor
+                type: PreProcessor
+                params:
+                clean_whitespace: true
+            - name: IndexTimeDocumentClassifier
+                type: TransformersDocumentClassifier
+                params:
+                batch_size: 16
+                use_gpu: false
+            - name: QueryTimeDocumentClassifier
+                type: TransformersDocumentClassifier
+                params:
+                use_gpu: false
+
+            pipelines:
+            - name: query_pipeline
+                nodes:
+                - name: Retriever
+                    inputs: [Query]
+                - name: Reader
+                    inputs: [Retriever]
+
+            - name: indexing_pipeline
+                nodes:
+                - name: PDFConverter
+                    inputs: [File]
+                - name: Preprocessor
+                    inputs: [PDFConverter]
+                - name: Retriever
+                    inputs: [Preprocessor]
+                - name: DocumentStore
+                    inputs: [Retriever]
+            """
+        f.write(yaml_content)
+    return path
+
+
+def test_indexing_pipeline_with_classifier():
+    # test correct load of indexing pipeline from yaml
+    pipeline = Pipeline.load_from_yaml(
+        SAMPLES_PATH / "pipelines" / "test.haystack-pipeline.yml", pipeline_name="indexing_pipeline"
+    )
+    pipeline.run(file_paths=SAMPLES_PATH / "pipelines" / "sample_pdf_1.pdf")
+    # test correct load of query pipeline from yaml
+    pipeline = Pipeline.load_from_yaml(
+        SAMPLES_PATH / "pipelines" / "test.haystack-pipeline.yml", pipeline_name="query_pipeline"
+    )
+    prediction = pipeline.run(
+        query="Who made the PDF specification?", params={"Retriever": {"top_k": 10}, "Reader": {"top_k": 3}}
+    )
+    assert prediction["query"] == "Who made the PDF specification?"
+    assert prediction["answers"][0].answer == "Adobe Systems"
+    assert prediction["answers"][0].meta["classification"]["label"] == "joy"
+    assert "_debug" not in prediction.keys()
+
+
+def test_query_pipeline_with_document_classifier():
+    # test correct load of indexing pipeline from yaml
+    pipeline = Pipeline.load_from_yaml(
+        SAMPLES_PATH / "pipelines" / "test.haystack-pipeline.yml", pipeline_name="indexing_pipeline"
+    )
+    pipeline.run(file_paths=SAMPLES_PATH / "pipelines" / "sample_pdf_1.pdf")
+    # test correct load of query pipeline from yaml
+    pipeline = Pipeline.load_from_yaml(
+        SAMPLES_PATH / "pipelines" / "test.haystack-pipeline.yml",
+        pipeline_name="query_pipeline_with_document_classifier",
```

Comment:
There is no pipeline with that name in the yaml. I'd expect an error if you manually run e2e/

Reply:
I think this might be a duplicate. I removed some tests using nearly identical pipelines, this must be one of those.